configs/eslint-config-compass/index.js

```diff
@@ -24,39 +25,46 @@ const reactConfigurations = [
 ];
 
 const testConfigurations = ['plugin:mocha/recommended'];
+
 const testRules = {
   'mocha/no-exclusive-tests': 'error',
   'mocha/no-hooks-for-single-case': 'off',
+  'mocha/no-setup-in-describe': 'off',
```

Comment:
We seem to disable this rule quite often, so I thought that it might make sense to just disable it. Anyone finds it useful?